src/styles/components/cards.scss

```
@@ -0,0 +1,113 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 166px;
+  padding: 32px 16px 16px;
+  background-color: $main-background-color;
+  border: 1px solid $card-border-color ;
+  border-radius: 5px;
+  transition: transform 300ms;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    padding-bottom: 4px;
+    font-size: $main-font-size;
+    line-height: 18px;
+    font-weight: 500;
+    color: $main-color-font;
+  }
+
+  &__code {
+    font-size: $small-font-size;
+    line-height: 14px;
+    font-weight: 400;
+    color: $card-secondary-font;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 16px 0 24px;
```

add margin-bottom: 16px to &__code. 
It's better use one way of setting margin for your elements
```suggestion
    margin-bottom: 24px;
```